packages/driver/test/cypress/integration/commands/actions/click_spec.js

```diff
@@ -257,25 +294,37 @@ describe('src/cy/commands/actions/click', () => {
     })
 
     it('will send all events even mousedown is defaultPrevented', () => {
-      const events = []
 
       const $btn = cy.$$('#button')
 
       $btn.get(0).addEventListener('mousedown', (e) => {
         e.preventDefault()
-
         expect(e.defaultPrevented).to.be.true
       })
 
-      _.each('mouseup click'.split(' '), (event) => {
-        return $btn.get(0).addEventListener(event, () => {
-          return events.push(event)
-        })
-      })
+      attachMouseClickListeners({ $btn })
 
-      cy.get('#button').click().then(() => {
-        expect(events).to.deep.eq(['mouseup', 'click'])
+      cy.get('#button').click().should('not.have.focus')
+
+      cy.getAll('$btn', 'pointerdown mousedown pointerup mouseup click').each(shouldBeCalled)
```

Comment:
need to think about a potentially better way to do this. not necessarily bad or wrong just need to discuss